Src/AutoFoqUnitTest/FoqMethodQueryTest.fs

```diff
@@ -87,6 +105,17 @@ let SelectMethodsDefineCorrectParameters (request: Type) =
                        (resultParameters |> Seq.toArray))) @>
     // Teardown
 
+[<Fact>]
+let ``Builder is correct`` () =
+    let expected =
+        { new ISpecimenBuilder with
+              member this.Create(r, c) = obj() }
+    let sut = FoqMethodQuery expected
+
+    let actual = sut.Builder
+
+    verify <@ expected = actual @>

```

Comment:
I've just noticed, by looking at the Pull Request, that this can be changed to `expected =? actual`.